softwareComponents/configuration/rofibot.cpp

```diff
@@ -2,10 +2,24 @@
 
 namespace rofi {
 
+double orientationToAngle( Orientation o ) {
+	switch ( o ) {
+		case Orientation::North:
+			return M_PI;
+		case Orientation::East:
+			return - M_PI_2;
+		case Orientation::South:
+			return 0;
+		case Orientation::West:
+			return M_PI_2;
+	}
+	assert( false && "Orientation was modified" );
+}
+
 void rofi::Module::setJointParams( int idx, const Joint::Positions& p ) {
     // Currently we invalidate all positions; ToDo: think if we can improve it
     assert( idx < _joints.size() );
-    _joints[ idx ].joint->positions = p;
+    std::transform( p.begin(), p.end(), _joints[ idx ].joint->positions.begin(), degToRad );
```

Comment:
Odkud pramení definice toho, že Positions předáváme jako stupně a nikoliv radiány? Toto mi přijde lehce matoucí pro uživatele - všechno je v radiánech, ale pozici kloubů předáváme jako stupně. Ocením i názor ostatních; např. @Ondraceq a @xondika

Reply:
Tady k tomuto bych se vrátil, až si tu novou konfiguraci zažijeme a budeme mít specifikovaný nový formát konfiguračních souborů. Teď tam máme stupně a taky stupně používáme při vytváření univerzálního modulu. Možná by nejlepší bylo umět obojí, ale to udělá zápis (trošku?) složitějším.

Reply:
Máme https://github.com/paradise-fi/RoFI/blob/503145cb2ba1926d4324b045970cb1b3cd636e43/softwareComponents/atoms/include/atoms/units.hpp#L5

A má to i literály

Reply:
Na to jsem zapomněl. Tak jsem to předělal tak, aby to ten Angle používalo. Takže je teď ~ všude ~ krom matic (`Matrix.h`), kde mi nepřijde rozumné přidávat další závislost (kvůli zpětné kompatibilitě by se ty funkce musely přetěžovat) a nemyslím si, že to za to stojí. Ale klidně se nechám přesvědčit.